lib/content_style.rb

```diff
@@ -1,118 +1,9 @@
 # frozen_string_literal: true
 
 require 'nokogiri'
-require 'parser'
+require 'content-style/parser'
+require 'content-style/linter'
```

Comment:
It's more common in gems to just `require` everything in this file, but nowhere else in the library. You do need to keep in mind the ordering of requires though.